source/documentation/common/upgrade/proc-Updating_the_Red_Hat_Virtualization_Manager.adoc

```diff
@@ -30,6 +30,11 @@ link:{URL_customer-portal}{URL_docs}{URL_lang-locale}{URL_product_rhv}4.3/html-s
 endif::4-3_local_db,4-3_remote_db,4-3_SHE[]
 ifdef::SHE_minor_updates,migrating_to_SHE,minor_updates[]
 link:{URL_virt_product_docs}{URL_format}installing_{URL_product_virt}_as_a_standalone_manager_with_local_databases/index#Enabling_the_Red_Hat_Virtualization_Manager_Repositories_install_RHVM[Enabling the {virt-product-fullname} {engine-name} Repositories] for {virt-product-fullname} 4.4.
++
+[NOTE]
+====
+If you are upgrading from {virt-product-shortname} version 4.4.0 through 4.4.6 to {virt-product-shortname} version 4.4.7 or later, you must add the EAP 7.4 channel to the list of subscription repositories `jb-eap-7.4-for-rhel-8-x86_64-rpms`, and following the upgrade, remove the `jb-eap-7.3-for-rhel-8-x86_64-rpms` from the lisat of subscription repositories.
```

Comment:
There is slight version mistmatch probably caused by several EAP 7.4 postpones, correct sentence should look like below:

If you are upgrading from {virt-product-shortname} version 4.4.0 through 4.4.8 to {virt-product-shortname} version 4.4.9 or later, you must add the EAP 7.4 channel to the list of subscription repositories `jb-eap-7.4-for-rhel-8-x86_64-rpms`, and following the upgrade, remove the `jb-eap-7.3-for-rhel-8-x86_64-rpms` from the lisat of subscription repositories.

Reply:
done